### PR TITLE
Change timetable

### PR DESCRIPTION
**Pull request by kwoon**

### CODE REVIEW

File: js/src/components/antd/generateConfig.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  isAfter,
+  isValidate,
+
+  locale: {
+    getWeekFirstDay: (locale) => parseLocale(locale).options.weekStartsOn,
+    getWeek: (locale, value) =>
+      getWeekYear(value, { locale: parseLocale(locale) }),
+
+    format: (locale, date, fmt) => {
+      // console.log(fmt);
```
Comment (Aliaksandrzaz) [inline js/src/components/antd/generateConfig.ts:68]: remove console.log

File: js/src/pages/UserDetails.tsx [hunk 2]
```diff
@@ -183,7 +183,6 @@ export const UserDetailsPage: React.FC = () => {
     <div className="human-resource-details">
       <Menu
         onClick={(e: any) => {
```
Comment (Aliaksandrzaz) [inline js/src/pages/UserDetails.tsx:185]: you can get "key" after restructuring, onClick={({key}) => {setSelected(key);}}

File: js/src/components/antd/generateConfig.ts (hunk 1 above)
Comment (kwoon) [reply js/src/components/antd/generateConfig.ts]: done

File: js/src/pages/UserDetails.tsx (hunk 2 above)
Comment (kwoon) [reply js/src/pages/UserDetails.tsx]: done

File: js/src/features/human-resouce/components/HumanResourceCalendar.tsx [hunk 3]
```diff
@@ -1,72 +1,162 @@
...
+};
+
+export const TimelineCell: React.FC<CellProps> = ({ date, absences = [] }) => {
+  const { className, title } = generateCellInfo(date, absences);
+
+  if (!className) {
+    return <td className="timetable-cell" />;
   }
 
-  return classes;
```
Comment (DmitryKus) [inline js/src/features/human-resouce/components/HumanResourceCalendar.tsx:43]: maybe we should separate this as a component? And `generateCellInfo` can be replace with variables assignment before render?

**Pull request merged**